lib/atomic/accounts/user.ex

```diff
@@ -79,6 +81,17 @@ defmodule Atomic.Accounts.User do
     |> unique_constraint(:email)
   end
 
+  defp validate_handle(changeset) do
+    changeset
+    |> validate_required([:handle])
+    |> validate_format(:handle, ~r/^[a-z0-9_]+$/,
+      message: "must only contain lowercase characters, numbers, and underscores"
```

Comment:
```suggestion
      message: "must only contain lowercase characters, numbers, and underscores"
```
It should also contain uppercases like twitter or github